src/components/Button/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import styles from './Button.module.css';
+
+const Button = ({ label, isActive = true, onClick }) => {
+    return (
+        <button className={`${styles.button} ${!isActive ? styles.inactive : ''}`} onClick={onClick}>
+            {label}
```

Comment:
as you use label as a children it would be better to use children props. It allows you pass not only string but any react components

Reply:
Changed label for children